djangocms_blog/models.py

```diff
@@ -232,7 +232,7 @@ def get_title(self):
         return title.strip()
 
     def get_keywords(self):
-        return self.safe_translation_getter('meta_keywords').strip().split(',')
+        return self.safe_translation_getter('meta_keywords', default='').strip().split(',')
```

Comment:
@fmarco So Django [doesn't save default values in DB schema](https://code.djangoproject.com/ticket/470). Means that all is good with migrations. I bet I am hitting some django-parler bug here. Maybe it doesn't consider field's default value in ```safe_translation_getter``` for some reason.